main.go

```diff
@@ -52,34 +55,113 @@ func fail(format string, v ...interface{}) {
 func main() {
 	var config Config
 	if err := stepconf.Parse(&config); err != nil {
-		fail("Issue with input: %s", err)
+		fail("issue with input: %s", err)
```

Comment:
these are error messages on the main, please make them back title case

Reply:
Fixed.